test/vmapi.test.js

```diff
@@ -14,11 +14,22 @@ var libuuid = require('libuuid');
 function uuid() {
     return (libuuid.create());
 }
+var async = require('async');
```

Comment:
Right. Actually, I don't need it, and it's a leftover from a previous implementation that used `whilst`, which AFAIK is not present in `vasync` unfortunately.